lib/requests/get-account-info-request.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const ACLChecker = require('../acl-checker')
+const AuthRequest = require('./auth-request')
+const { actualSize } = require('../utils')
+const $rdf = require('rdflib')
+const HTTPError = require('../http-error')
+const LRU = require('lru-cache')
+
+const PIM = $rdf.Namespace('http://www.w3.org/ns/pim/space#')
+const SOLID = $rdf.Namespace('http://www.w3.org/ns/solid/terms#')
+
+const cache = new LRU({
```

Comment:
I see the benefit of caching, but this could make this quite annoying for users.

It may be possible that the user's quota is at 100%, but that the view does not resemble this yet. We may be getting bug reports because of this.

So either we should remove this cache, or we should add a cache invalidation step every time a file system change happens in the user's directories.

Reply:
Removed cache in 86a33e536aa8398cc4cd8b8aad650e26e0b67a33